src/deepsparse/transformers/schemas/text_generation_schemas.py

```diff
@@ -22,20 +23,21 @@
 
 
 # Based off of https://huggingface.co/docs/transformers/main_classes/text_generation#transformers.GenerationConfig # noqa E501
+@dataclass
```

Comment:
Any reason to use data classes over Pydantic BaseModels? Specially because we're already using BaseModel throughout the codebase?

Reply:
doesnt seem like it needs to be a pydantic model if only used internally and just holds default data